composer.json

```diff
@@ -19,15 +19,15 @@
         "doctrine/annotations": "^1.6",
         "liip/metadata-parser": "^0.2",
         "pnz/json-exception": "^0.1.0",
```

Comment:
I guess we'd also want to switch this to safe/ like in our projects.

Reply:
agree, it is the more commonly known library, even though the author of pnz/json-exception is well known :-D

Reply:
I understand your reasons, but I would keep the current `pnz/json-exception` library.
My reason is the following: the `safe` library is pulling in 1.2Mb, of which we would use the JSON exception handling, while the `pnz` library is doing the same in only 48Kb

Reply:
Do you plan to release a stable version of your library then?

Reply:
Also I'm pretty sure we use other functions in liip/serializer that should use the safe alternative in theory.

Reply:
https://github.com/thePanz/json-exception/issues/2 is blocking an update

Reply:
@thePanz please update this to use your stable tag